R/tweak-homepage.R

```diff
@@ -48,12 +48,13 @@ tweak_sidebar_html <- function(html, sidebar = TRUE, show_badges = TRUE) {
     return(html)
   }
 
+  # this extracts *and removes* badges from HTML
```

Comment:
I am mislead by the name "extract" actually. I looked at synonyms and maybe "pull out" would make it clearer it removes badges. In theory "extract" should however be clear so it might be my fault!